content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -162,6 +162,8 @@ Remove midclt call from 24.10.1 known issues if [NAS-132914](https://ixsystems.a
 iXsystems is pleased to release TrueNAS 24.10.1!
 This is a maintenance release and includes refinement and fixes for issues discovered after the 24.10.0 and 24.10.0.X releases.
 
+* sendmail is removed for security reasons ([NAS-133174](https://ixsystems.atlassian.net/browse/NAS-133174)). Users previously using sendmail as part of their custom scripting should use `exim` instead.
+* The boot pool is now properly enforcing the default `setuid` and `noexec` options ([NAS-127825](https://ixsystems.atlassian.net/browse/NAS-127825)). This restores the default boot pool behavior to be restricted from general use.
```

Comment:
Possibly include a user instruction here? If they were previously storing exec scripts on the admin /home (on boot pool) then they need to move them to a data pool